layout/_partial/header.ejs

```diff
@@ -1,18 +1,18 @@
 <header id="header">
-  <a href="<%- url_for("/") %>">
+  <a class="u-url" href="<%- url_for("/") %>">
   <% if (theme.logo && theme.logo.enabled) { %>
     <% if (theme.gravatar && (theme.gravatar.email || theme.gravatar.hash) && theme.logo.gravatar) { %>
       <% if (theme.gravatar.email) { %>
-        <div id="logo" style="background-image: url(<%- gravatar(theme.gravatar.email) %>);"></div>
+        <img id="logo" class="u-logo" src="<%- gravatar(theme.gravatar.email) %>" />
```

Comment:
I had to replace the `div`+ CSS `background-image` with a proper `img` tag, as the validator wasn't able to find the image URL otherwise. It doesn't seem to have any impact on the final layout (I tested locally with the default logo as well as a gravatar logo).